.github/workflows/ci_log_subscriber.yml

```
@@ -0,0 +1,43 @@
+on:
+  workflow_call:
+env:
+  GO_VERSION: "1.22"
+
+jobs:
+  ci-log-subscriber-lint:
+    runs-on: ubuntu-latest
+    if: github.event_name != 'push' || !startsWith(github.event.head_commit.message, 'v')
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)